spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiAutoConfigurationTest.java

```diff
@@ -58,7 +58,9 @@ public class AdminServerUiAutoConfigurationTest implements WithAssertions {
 	public class ReactiveUiConfigurationTest {
 
 		private final ReactiveWebApplicationContextRunner contextRunner = new ReactiveWebApplicationContextRunner()
-				.withPropertyValues("--spring.boot.admin.ui.available-languages=de", "--spring.webflux.base-path=test")
+				.withPropertyValues("--spring.boot.admin.ui.available-languages=de", "--spring.webflux.base-path=test",
+						"--spring.boot.admin.contextPath=test", "--spring.boot.admin.ui.additional-routes[0]=/info/**",
```

Comment:
SBA context path is used for filtering in case of webflux application. I noticed that when added my testcases before implementation (tests didn't fail).
I'm not sure whether it's a bug or intended behaviour, so I leave the auto configuration logic as is.

Reply:
I suppose, I should wait for that PR to be merged, rebase mine and then merge. Will that be ok?

Reply:
yes sounds good

Reply:
Done